compiler/src/iree/compiler/Dialect/VMVX/Transforms/ResolveBufferDescriptors.cpp

```diff
@@ -335,6 +336,47 @@ getBaseBufferReplacementForDescriptor(GetBufferDescriptorOp descriptorOp,
       .getResult(0);
 }
 
+struct FromMemRefAssumeAlignment
+    : public OpRewritePattern<GetBufferDescriptorOp> {
+  using OpRewritePattern<GetBufferDescriptorOp>::OpRewritePattern;
+  LogicalResult matchAndRewrite(GetBufferDescriptorOp op,
+                                PatternRewriter &rewriter) const override {
+    auto assumeOp = op.getSource().getDefiningOp<memref::AssumeAlignmentOp>();
+    if (!assumeOp) {
+      return failure();
+    }
+    auto binding = assumeOp.getMemref()
+                       .getDefiningOp<IREE::HAL::InterfaceBindingSubspanOp>();
+    if (!binding) {
+      return failure();
+    }
+
+    auto loc = op.getLoc();
```

Comment:
nit:
```suggestion
    Location loc = op.getLoc();
```

Reply:
ah, it is the issue from copy-paste, fixed. I'll update the other place when I refactor this file.